build.psm1

```diff
@@ -220,6 +220,10 @@ function Start-PSBuild {
         [switch]$NoPSModuleRestore,
```

Comment:
@rjmholt, your last commit had 7 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

New-CimSession.A cim session can be created
```powershell
The term 'Remove-CimSession' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/CimCmdlets/CimSession.Tests.ps1: line 14
14:                 $sessions | Remove-CimSession -ErrorAction SilentlyContinue

```
New-CimSession.A Cim session can be retrieved
```powershell
The term 'Remove-CimSession' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/CimCmdlets/CimSession.Tests.ps1: line 14
14:                 $sessions | Remove-CimSession -ErrorAction SilentlyContinue

```
New-CimSession.A cim session can be removed
```powershell
The term 'Remove-CimSession' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/CimCmdlets/CimSession.Tests.ps1: line 14
14:                 $sessions | Remove-CimSession -ErrorAction SilentlyContinue

```
Get-CimClass.can get CIM_Error CIM class
```powershell
The term 'Get-CimClass' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/CimCmdlets/Get-CimClass.Tests.ps1: line 12
12:             Get-CimClass -ClassName CIM_Error | Should -Not -BeNullOrEmpty

```
Get-CimClass.can get class when namespace is specified
```powershell
The term 'Get-CimClass' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/CimCmdlets/Get-CimClass.Tests.ps1: line 16
16:             Get-CimClass -ClassName CIM_OperatingSystem -Namespace root/cimv2 | Should -Not -BeNullOrEmpty

```